Geometry/VeryForwardGeometry/python/geometryRP_cfi.py

```diff
@@ -81,8 +81,38 @@
         'Geometry/VeryForwardData/data/RP_Cuts_Per_Region.xml',
         'Geometry/VeryForwardData/data/RP_Param_Beam_Region.xml')
 
+ctppsDiamondGeomXMLFiles = cms.vstring(
```

Comment:
@slava77 - I don't think we need to re-organize xml files in older releases. These files are present in 80X.